sefaria/helper/topic.py

```diff
@@ -1042,81 +1042,79 @@ def topic_change_category(topic_obj, new_category, old_category="", rebuild=Fals
         rebuild_topic_toc(topic_obj, category_changed=True)
     return topic_obj
 
-def update_topic_titles(topic_obj, data):
-    for lang in ['en', 'he']:
-        for title in topic_obj.get_titles(lang):
-            topic_obj.remove_title(title, lang)
-        for title in data['altTitles'][lang]:
-            topic_obj.add_title(title, lang)
-
-    topic_obj.add_title(data['title'], 'en', True, True)
-    topic_obj.add_title(data['heTitle'], 'he', True, True)
-    return topic_obj
-
-
-def update_authors_place_and_time(topic_obj, data, dataSource='learning-team-editing-tool'):
+def update_topic_titles(topic, **kwargs):
```

Comment:
My idea of using `**data` here was that so that we could use explicit named parameters here which makes the function easier to understand. However, in many of the other cases in this PR where kwargs are used it seems fine because we don't know necessarily what the keys of the kwargs will be.

In summary: use kwargs when the keys are dynamic. Use named parameters when they're static so users can more easily understand how to call the function.